### PR TITLE
Add tru

### PR DESCRIPTION
**Pull request by sungwoncho**
Hi, thanks for creating this helpful list. This PR adds a new language called tru.

* https://github.com/sungwoncho/tru
* https://esolangs.org/wiki/Tru

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,7 @@ Curated list of awesome esoteric programming languages, resources and related pr
 * [Spooky](https://spookylang.com) - The scariest programming language  :jack_o_lantern: :skull:.
 * [Suzy](https://github.com/gvx/suzy) - 3D language inspired in Befunge.
 * [Thue](https://github.com/jcolag/Thue) - *Matrioshka* language based on nondeterministic string rewriting of a semi-Thue grammar.
+* [Tru](https://github.com/sungwoncho/tru) -  A stack-based language with prefix-free brackets
```
Comment (angrykoala) [inline README.md:64]: The link and description look fine, there are some minor format issues though.

The separator between link and description has an extra space before `A stack-based`.

The description should end with `.`

**Comment by angrykoala** [comment]
Hi @sungwoncho thanks for your PR.

The language seems interesting and the description is good, I've added a comment regarding some format issues that need to be addressed before merging

File: README.md (hunk 1 above)
Comment (sungwoncho) [reply README.md]: Thanks for the correction.

**Comment by sungwoncho** [comment]
@angrykoala thanks for your review.

**Pull request merged**